server/src/internalClusterTest/java/org/elasticsearch/search/query/MultiMatchQueryIT.java

```diff
@@ -601,6 +601,201 @@ public void testCrossFieldMode() throws ExecutionException, InterruptedException
         assertFirstHit(searchResponse, hasId("nowHero"));
     }
 
+    public void testCombinedFieldModeFieldGrouping() {
+        SearchResponse searchResponse = client().prepareSearch("test")
+            .setQuery(randomizeType(multiMatchQuery("captain america", "full_name", "first_name", "last_name")
```

Comment:
do we need randomizeType here? Looks like it doesn't do anything useful for Type.COMBINED_FIELDS